test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -2102,6 +2128,34 @@ TEST_P(XdsRbacTestWithActionPermutations, EmptyRbacPolicy) {
       grpc::StatusCode::PERMISSION_DENIED);
 }
 
+TEST_P(XdsRbacTestWithActionPermutations,
```

Comment:
Please move all of the new tests in this suite down to line 2978, so that all of the audit logging tests are after all of the tests that cover the actual authorization flow.